Index: xen-4.3.0~rc4+38+gb03165b/tools/debugger/gdbsx/xg/xg_main.c
===================================================================
--- xen-4.3.0~rc4+38+gb03165b.orig/tools/debugger/gdbsx/xg/xg_main.c	2013-06-16 21:22:09.000000000 +0200
+++ xen-4.3.0~rc4+38+gb03165b/tools/debugger/gdbsx/xg/xg_main.c	2013-06-25 16:15:05.097364124 +0200
@@ -179,7 +179,7 @@ _domctl_hcall(uint32_t cmd,            /
     hypercall.op = __HYPERVISOR_domctl;
     hypercall.arg[0] = (unsigned long)&domctl;
 
-    rc = ioctl(_dom0_fd, IOCTL_PRIVCMD_HYPERCALL, (ulong)&hypercall);
+    rc = ioctl(_dom0_fd, IOCTL_PRIVCMD_HYPERCALL, (unsigned long)&hypercall);
     if (domctlarg && sz)
         munlock(domctlarg, sz);
     return rc;
@@ -219,7 +219,7 @@ _check_hyp(int guest_bitness)
     hypercall.arg[0] = (unsigned long)XENVER_capabilities;
     hypercall.arg[1] = (unsigned long)&xen_caps;
 
-    rc = ioctl(_dom0_fd, IOCTL_PRIVCMD_HYPERCALL, (ulong)&hypercall);
+    rc = ioctl(_dom0_fd, IOCTL_PRIVCMD_HYPERCALL, (unsigned long)&hypercall);
     munlock(&xen_caps, sizeof(xen_caps));
     XGTRC("XENCAPS:%s\n", xen_caps);
 
